tasks/agent.py

```diff
@@ -450,6 +450,10 @@ def omnibus_build(
                 env['SIGN_PFX'] = "{}".format(pfxfile)
                 env['SIGN_PFX_PW'] = "{}".format(pfxpass)
 
+            if sys.platform == 'win32' and (release_version == 'nightly' or release_version == 'nightly-a7'):
+                # Produce debug installer on Windows for nightlies
+                env["DEBUG_CUSTOMACTION"] = 'true'
```

Comment:
I think this is something we should manually enable only, not make it automatic for nightly builds. Saying so because debug builds are significantly different than release builds (eg: msvc initializes all uninitialized variables to 0, among other protections) and a behavior change between two builds (nightly and non-nightly) might be really puzzling without knowing this default is here.